diff --git a/browser/components/preferences/preferences.js b/browser/components/preferences/preferences.js
index e95831780d00111f27bb4ad78a6ed8ec39cf886e..b1a2053d50c277beabdd2ba737b1b58d89afb098 100644
--- a/browser/components/preferences/preferences.js
+++ b/browser/components/preferences/preferences.js
@@ -583,6 +583,11 @@ function init_all() {
 
   let categories = document.getElementById("categories");
   categories.addEventListener("change-view", event => {
+    if (event.target.view === "paneMidoriCenter") {
+      let mainWindow = window.browsingContext.topChromeWindow;
+      mainWindow.openTrustedLinkIn("about:center", "tab");
+      return;
+    }
     gotoPref(event.target.view);
   });
 
diff --git a/browser/components/preferences/preferences.xhtml b/browser/components/preferences/preferences.xhtml
index 9760a4a35b0b3bd21edec07a70c10bccc23e4a09..b452a41071c9a9aef869c3e2816b5e21d79db90d 100644
--- a/browser/components/preferences/preferences.xhtml
+++ b/browser/components/preferences/preferences.xhtml
@@ -200,6 +200,11 @@
         iconsrc="chrome://browser/skin/preferences/mozilla-16.svg"
         data-l10n-id="more-from-moz-title2">
       </html:moz-page-nav-button>
+  <html:moz-page-nav-button id="category-midori-center"
+    view="paneMidoriCenter"
+    iconsrc="chrome://browser/skin/preferences/category-general.svg"
+    data-l10n-id="pane-midori-center-title">
+  </html:moz-page-nav-button>
 
       <!-- Hidden sub-page entries (CSS display:none via .hidden-category).
            These keep gotoPref() routing working for sub-pages that are

