diff --git a/browser/components/preferences/preferences.js b/browser/components/preferences/preferences.js
index 3c60cc9f199ba5836ea1233fbbeb74a6dd63b0c0..14dc238afe9f5b1d8f2121639e66c6ff68206ede 100644
--- a/browser/components/preferences/preferences.js
+++ b/browser/components/preferences/preferences.js
@@ -396,7 +396,15 @@ function init_all() {
   gMainPane.preInit();
 
   let categories = document.getElementById("categories");
-  categories.addEventListener("select", event => gotoPref(event.target.value));
+  categories.addEventListener("select", event => {
+    if (event.target.value === "paneMidoriCenter") {
+      let mainWindow = window.browsingContext.topChromeWindow;
+      mainWindow.openTrustedLinkIn("about:center", "tab");
+      categories.clearSelection();
+      return;
+    }
+    gotoPref(event.target.value);
+  });
 
   document.documentElement.addEventListener("keydown", function (event) {
     if (event.keyCode == KeyEvent.DOM_VK_TAB) {
diff --git a/browser/components/preferences/preferences.xhtml b/browser/components/preferences/preferences.xhtml
index b006d5e5e64c418fbf930c010401e4ea3a3647a3..3b5d2a3f0a17aa55ab462a0843760f8de8bcbb20 100644
--- a/browser/components/preferences/preferences.xhtml
+++ b/browser/components/preferences/preferences.xhtml
@@ -196,6 +196,15 @@
           <label class="category-name" flex="1" data-l10n-id="more-from-moz-title"></label>
         </richlistitem>
 
+        <richlistitem id="category-midori-center"
+                      class="category"
+                      value="paneMidoriCenter"
+                      align="center"
+                      tooltiptext="Midori Center">
+          <image class="category-icon" style="list-style-image: url('chrome://browser/skin/preferences/category-general.svg');"/>
+          <label class="category-name" flex="1">Midori Center</label>
+        </richlistitem>
+
         <!-- The following <richlistitem> elements are hidden from the list of items via the hidden-category class.
             The intent is that they act as sub-pages, accessible from the UI of one of the main pages, but are
             not listed as primary pages themselves via the selector.

