diff --git a/browser/components/preferences/search.js b/browser/components/preferences/search.js
index d8a36e15da281872d4be9a8d46176ad785f8c82d..9d39523912620f06f038cff072eb0ceecd67c6bb 100644
--- a/browser/components/preferences/search.js
+++ b/browser/components/preferences/search.js
@@ -926,7 +926,7 @@ Preferences.addSetting(
           items: [
             {
               id: editId,
-              l10nId: "search-edit-engine-2",
+              l10nId: "search-edit-engine",
               control: "moz-button",
               iconSrc: "chrome://global/skin/icons/edit-outline.svg",
               slot: "actions",
@@ -1036,6 +1036,7 @@ var gSearchPane = {
     initSettingGroup("searchSuggestions");
     initSettingGroup("firefoxSuggest");
     initSettingGroup("searchShortcuts");
+    this.disableSearchEngineEditingUI();
     this._engineStore = new EngineStore();
     gEngineView = new EngineView(this._engineStore);
 
@@ -1061,6 +1062,32 @@ var gSearchPane = {
     });
   },
 
+  disableSearchEngineEditingUI() {
+    let defaultEngine = document.getElementById("defaultEngine");
+    if (defaultEngine) {
+      defaultEngine.disabled = true;
+    }
+
+    let defaultPrivateEngine = document.getElementById("defaultPrivateEngine");
+    if (defaultPrivateEngine) {
+      defaultPrivateEngine.disabled = true;
+    }
+
+    let separatePrivateToggle = document.getElementById(
+      "browserSeparateDefaultEngine"
+    );
+    if (separatePrivateToggle) {
+      separatePrivateToggle.disabled = true;
+    }
+
+    let oneClickSearchProvidersGroup = document.getElementById(
+      "oneClickSearchProvidersGroup"
+    );
+    if (oneClickSearchProvidersGroup) {
+      oneClickSearchProvidersGroup.hidden = true;
+    }
+  },
+
   // ADDRESS BAR
   handleEvent(aEvent) {
     if (aEvent.type != "command") {
