diff --git a/browser/components/preferences/search.js b/browser/components/preferences/search.js
index 07b6fe86a5d1009924f647e05c247b30e03d0249..19d19292ca94f509ea4482c9c47970eb4e5645b1 100644
--- a/browser/components/preferences/search.js
+++ b/browser/components/preferences/search.js
@@ -49,6 +49,7 @@ var gSearchPane = {
     initSettingGroup("defaultEngine");
     initSettingGroup("searchSuggestions");
     initSettingGroup("firefoxSuggest");
+    this.disableSearchEngineEditingUI();
     this._engineStore = new EngineStore();
     gEngineView = new EngineView(this._engineStore);
 
@@ -78,6 +79,32 @@ var gSearchPane = {
     lazy.separatePrivateDefaultPrefValue;
   },
 
+  disableSearchEngineEditingUI() {
+    let defaultEngine = document.getElementById("defaultEngine");
+    if (defaultEngine) {
+      defaultEngine.disabled = true;
+    }
+
+    let defaultPrivateEngine = document.getElementById("defaultPrivateEngine");
+    if (defaultPrivateEngine) {
+      defaultPrivateEngine.disabled = true;
+    }
+
+    let separatePrivateToggle = document.getElementById(
+      "browserSeparateDefaultEngine"
+    );
+    if (separatePrivateToggle) {
+      separatePrivateToggle.disabled = true;
+    }
+
+    let oneClickSearchProvidersGroup = document.getElementById(
+      "oneClickSearchProvidersGroup"
+    );
+    if (oneClickSearchProvidersGroup) {
+      oneClickSearchProvidersGroup.hidden = true;
+    }
+  },
+
   // ADDRESS BAR
   handleEvent(aEvent) {
     if (aEvent.type != "command") {
