diff --git a/browser/base/content/navigator-toolbox.js b/browser/base/content/navigator-toolbox.js
index 15469e9d9b..ff22baa5b8 100644
--- a/browser/base/content/navigator-toolbox.js
+++ b/browser/base/content/navigator-toolbox.js
@@ -40,7 +40,6 @@ document.addEventListener(
 
     function onCommand(event) {
       let element = event.target.closest(`
-        #firefox-view-button,
         .content-analysis-indicator,
         #bookmarks-toolbar-button,
         #PlacesToolbar,
@@ -55,10 +54,6 @@ document.addEventListener(
       }
 
       switch (element.id) {
-        case "firefox-view-button":
-          FirefoxViewHandler.openTab();
-          break;
-
         case "bookmarks-toolbar-button":
           PlacesToolbarHelper.onPlaceholderCommand();
           break;
@@ -103,7 +98,6 @@ document.addEventListener(
 
     function onMouseDown(event) {
       let element = event.target.closest(`
-        #firefox-view-button,
         #alltabs-button,
         #pageActionButton,
         #downloads-button,
@@ -116,10 +110,6 @@ document.addEventListener(
       }
 
       switch (element.id) {
-        case "firefox-view-button":
-          FirefoxViewHandler.openToolbarMouseEvent(event);
-          break;
-
         case "alltabs-button":
           gTabsPanel.showAllTabsPanel(event, "alltabs-button");
           break;
