diff --git a/browser/base/content/navigator-toolbox.js b/browser/base/content/navigator-toolbox.js
index be588fafee0202d0bc17f5a600e7d147830ea3d1..c93999aff58cb6f85a5545c9cdb1648796865354 100644
--- a/browser/base/content/navigator-toolbox.js
+++ b/browser/base/content/navigator-toolbox.js
@@ -44,7 +44,6 @@ document.addEventListener(
 
     function onCommand(event) {
       let element = event.target.closest(`
-        #firefox-view-button,
         .content-analysis-indicator,
         .private-browsing-indicator-button,
         #bookmarks-toolbar-button,
@@ -60,10 +59,6 @@ document.addEventListener(
       }
 
       switch (element.id) {
-        case "firefox-view-button":
-          FirefoxViewHandler.openTab();
-          break;
-
         case "bookmarks-toolbar-button":
           PlacesToolbarHelper.onPlaceholderCommand();
           break;
@@ -129,7 +124,6 @@ document.addEventListener(
 
     function onMouseDown(event) {
       let element = event.target.closest(`
-        #firefox-view-button,
         #alltabs-button,
         #pageActionButton,
         #downloads-button,
@@ -142,10 +136,6 @@ document.addEventListener(
       }
 
       switch (element.id) {
-        case "firefox-view-button":
-          FirefoxViewHandler.openToolbarMouseEvent(event);
-          break;
-
         case "alltabs-button":
           gTabsPanel.showAllTabsPanel(event, "alltabs-button");
           break;
