diff --git a/browser/base/content/navigator-toolbox.js b/browser/base/content/navigator-toolbox.js
index 362a85c20d8f0633fa78f85578e4cf015fdd0e44..5ecffb8b6683169d78cf26e9599bce83325e2525 100644
--- a/browser/base/content/navigator-toolbox.js
+++ b/browser/base/content/navigator-toolbox.js
@@ -45,7 +45,6 @@ document.addEventListener(
 
     function onCommand(event) {
       let element = event.target.closest(`
-        #firefox-view-button,
         .content-analysis-indicator,
         #bookmarks-toolbar-button,
         #PlacesToolbar,
@@ -60,10 +59,6 @@ document.addEventListener(
       }
 
       switch (element.id) {
-        case "firefox-view-button":
-          FirefoxViewHandler.openTab();
-          break;
-
         case "bookmarks-toolbar-button":
           PlacesToolbarHelper.onPlaceholderCommand();
           break;
@@ -108,7 +103,6 @@ document.addEventListener(
 
     function onMouseDown(event) {
       let element = event.target.closest(`
-        #firefox-view-button,
         #alltabs-button,
         #pageActionButton,
         #downloads-button,
@@ -121,10 +115,6 @@ document.addEventListener(
       }
 
       switch (element.id) {
-        case "firefox-view-button":
-          FirefoxViewHandler.openToolbarMouseEvent(event);
-          break;
-
         case "alltabs-button":
           gTabsPanel.showAllTabsPanel(event, "alltabs-button");
           break;
