diff --git a/browser/base/content/navigator-toolbox.js b/browser/base/content/navigator-toolbox.js
index 1982ebadbd2df5ea309db487a4f30288f1af43e5..ca4569bd33bf1bf9ab086fd1dddd615a63370614 100644
--- a/browser/base/content/navigator-toolbox.js
+++ b/browser/base/content/navigator-toolbox.js
@@ -44,7 +44,6 @@ document.addEventListener(
 
     function onCommand(event) {
       let element = event.target.closest(`
-        #firefox-view-button,
         .content-analysis-indicator,
         #bookmarks-toolbar-button,
         #PlacesToolbar,
@@ -59,10 +58,6 @@ document.addEventListener(
       }
 
       switch (element.id) {
-        case "firefox-view-button":
-          FirefoxViewHandler.openTab();
-          break;
-
         case "bookmarks-toolbar-button":
           PlacesToolbarHelper.onPlaceholderCommand();
           break;
@@ -107,7 +102,6 @@ document.addEventListener(
 
     function onMouseDown(event) {
       let element = event.target.closest(`
-        #firefox-view-button,
         #alltabs-button,
         #pageActionButton,
         #downloads-button,
@@ -120,10 +114,6 @@ document.addEventListener(
       }
 
       switch (element.id) {
-        case "firefox-view-button":
-          FirefoxViewHandler.openToolbarMouseEvent(event);
-          break;
-
         case "alltabs-button":
           gTabsPanel.showAllTabsPanel(event, "alltabs-button");
           break;
