From e8edecb24d566451e9c5053dc64cf48aba374393 Mon Sep 17 00:00:00 2001
From: Alfonso Hernandez <alfonso.hernandez@astian.org>
Date: Tue, 18 Nov 2025 08:29:26 -0500
Subject: [PATCH] include midori custom page welcome

---
 browser/components/BrowserGlue.sys.mjs | 29 ++++++++++++++++++++++++++
 1 file changed, 29 insertions(+)

diff --git a/browser/components/BrowserGlue.sys.mjs b/browser/components/BrowserGlue.sys.mjs
index f1cabb13ca..b6cdedc6f6 100644
--- a/browser/components/BrowserGlue.sys.mjs
+++ b/browser/components/BrowserGlue.sys.mjs
@@ -1672,6 +1672,23 @@ BrowserGlue.prototype = {
   },
 
   async _maybeShowDefaultBrowserPrompt() {
+
+    // =========================================================================
+    // Midori Browser Welcome
+
+    const MIDORI_WELCOME_PREF = 'midori.welcome.enabled';
+    const MidoriSeenPref = 'midori.welcome.seen';
+
+    // NOTE: Get bool prefs takes in two options, the pref string and a default
+    // value if it is not defined
+
+    const welcomeEnabled = Services.prefs.getBoolPref(MIDORI_WELCOME_PREF, true);
+    const welcomeSeen = Services.prefs.getBoolPref(MidoriSeenPref, false);
+
+    if (welcomeEnabled && !welcomeSeen)
+      // Open the welcome page in a new tab
+      this._showWelcomeDialog();
+    
     // Ensuring the user is notified arranges the following ordering.  Highest
     // priority is datareporting policy modal, if present.  Second highest
     // priority is the upgrade dialog, which can include a "primary browser"
@@ -1776,4 +1793,16 @@ BrowserGlue.prototype = {
     "nsIObserver",
     "nsISupportsWeakReference",
   ]),
+
+  /**
+  * Displays the welcome dialog. Triggered \`_maybeShowDefaultBrowserPrompt\`.
+  *
+  * @todo Add gluon docs for how to show custom popups on startup
+  * @patch Pulse Browser
+  */
+  _showWelcomeDialog() {
+    lazy.BrowserWindowTracker.getTopWindow().gDialogBox.open(
+   "chrome://browser/content/welcome/welcome.html"
+    );
+  },
 };
-- 
2.51.1