diff --git a/browser/components/moz.build b/browser/components/moz.build
index 913558d43c..16defb5f32 100644
--- a/browser/components/moz.build
+++ b/browser/components/moz.build
@@ -80,6 +80,9 @@ DIRS += [
     "webrtc",
 ]
 
+# Pulse: Custom components
+DIRS += ["autohide", "center", "gradient", "memory", "msidebar", "tor", "verticaltabs", "welcome", "workspace"]
+
 DIRS += ["build"]
 
 if CONFIG["MOZ_SELECTABLE_PROFILES"]:
