diff --git a/browser/components/moz.build b/browser/components/moz.build
index 913558d43c8c19f162feb2e08c48676f86e9cac2..972a7dd40971ba9a99a8206ae11ace516ec86495 100644
--- a/browser/components/moz.build
+++ b/browser/components/moz.build
@@ -80,6 +80,9 @@ DIRS += [
     "webrtc",
 ]
 
+# Pulse: Custom components
+DIRS += ["memory", "welcome"]
+
 DIRS += ["build"]
 
 if CONFIG["MOZ_SELECTABLE_PROFILES"]:
