diff --git a/browser/components/moz.build b/browser/components/moz.build
index 4a4b96b779f33b17082cf79563821ae1164c1dd1..4387fc12cb9e2e34bc23a0af5306911bbd8d5873 100644
--- a/browser/components/moz.build
+++ b/browser/components/moz.build
@@ -82,6 +82,9 @@ DIRS += [
     "webrtc",
 ]
 
+# Midori custom components
+DIRS += ["autohide", "center", "cws", "gradient", "lifecycle", "memory", "msidebar", "profilemigration", "setup", "shortcuts", "tabprotect", "tabsleep", "verticaltabs", "webapps", "workspace"]
+
 DIRS += ["build"]
 
 if CONFIG["MOZ_SELECTABLE_PROFILES"]:
