diff --git a/browser/components/moz.build b/browser/components/moz.build
index 913558d43c8c19f162feb2e08c48676f86e9cac2..8bcf7a23008d7ad266d0126957c84b2c35560ea6 100644
--- a/browser/components/moz.build
+++ b/browser/components/moz.build
@@ -80,6 +80,9 @@ DIRS += [
     "webrtc",
 ]
 
+# Pulse: Custom components
+DIRS += ["autohide", "center", "gradient", "memory", "msidebar", "setup", "tor", "verticaltabs", "welcome", "workspace"]
+
 DIRS += ["build"]
 
 if CONFIG["MOZ_SELECTABLE_PROFILES"]:
