diff --git a/browser/components/moz.build b/browser/components/moz.build
index 63b59082d8c8be455530f81ed19d8fac49937200..41ec7fde77fb039cba1cf4275d7a01753b9eee99 100644
--- a/browser/components/moz.build
+++ b/browser/components/moz.build
@@ -80,6 +80,9 @@ DIRS += [
     "webrtc",
 ]
 
+# Pulse: Custom components
+DIRS += ["autohide", "center", "cws", "gradient", "memory", "msidebar", "setup", "shortcuts", "tabsleep", "tor", "verticaltabs", "workspace"]
+
 DIRS += ["build"]
 
 if CONFIG["MOZ_SELECTABLE_PROFILES"]:
