diff --git a/browser/components/moz.build b/browser/components/moz.build
index 799e5246e6d11a3725129975154f524b8fe00a16..ef851ef46ad2446bfe31a15568aa8a9574194abc 100644
--- a/browser/components/moz.build
+++ b/browser/components/moz.build
@@ -81,6 +81,9 @@ DIRS += [
     "webrtc",
 ]
 
+# Midori custom components
+DIRS += ["autohide", "center", "cws", "gradient", "lifecycle", "memory", "msidebar", "profilemigration", "setup", "shortcuts", "tabsleep", "verticaltabs", "webapps", "workspace"]
+
 DIRS += ["build"]
 
 if CONFIG["MOZ_SELECTABLE_PROFILES"]:
