diff --git a/browser/app/profile/firefox.js b/browser/app/profile/firefox.js
index 2b852e0225accd7c97961cff1eb817bed5a2eb19..be258a1df6fac84d548e89b4b0d8a492caa2667b 100644
--- a/browser/app/profile/firefox.js
+++ b/browser/app/profile/firefox.js
@@ -1399,7 +1399,7 @@ pref("browser.preferences.experimental.hidden", false);
 // Whether we show the "More from Mozilla" section.
 pref("browser.preferences.moreFromMozilla", true);
 // Whether we show the "AI Controls" pane.
-pref("browser.preferences.aiControls", true);
+pref("browser.preferences.aiControls", false);
 // Whether to show unavailable AI controls regardless of region/locale
 // restrictions. This is intended for localizers to be able to see the strings.
 pref("browser.preferences.aiControls.showUnavailable", false);
@@ -3545,6 +3545,8 @@ pref("toolkit.contentRelevancy.ingestEnabled", false);
 // Pref to enable extra logging for the content relevancy feature
 pref("toolkit.contentRelevancy.log", false);
 
+#include midori-browser.js
+
 // The number of days after which to rotate the context ID. 0 means to disable
 // rotation altogether.
 pref("browser.contextual-services.contextId.rotation-in-days", 3);
