diff --git a/devtools/client/aboutdebugging/src/actions/runtimes.js b/devtools/client/aboutdebugging/src/actions/runtimes.js
index 52148f80b2803d75f2f1b84f8379389b68e66a65..1243fe7663a3a719bf927e4547428e1e5645a1ae 100644
--- a/devtools/client/aboutdebugging/src/actions/runtimes.js
+++ b/devtools/client/aboutdebugging/src/actions/runtimes.js
@@ -60,6 +60,10 @@ const CONNECTION_TIMING_OUT_DELAY = 3000;
 const CONNECTION_CANCEL_DELAY = 13000;
 
 async function getRuntimeIcon(runtime, channel) {
+  if (runtime.type === RUNTIMES.THIS_FIREFOX) {
+    return "chrome://branding/content/about-logo.svg";
+  }
+
   if (runtime.isFenix) {
     switch (channel) {
       case "release":
