diff --git a/browser/actors/ContextMenuChild.sys.mjs b/browser/actors/ContextMenuChild.sys.mjs
index bdb925257894ad76f7eaab179f41847e3f1d7d33..f16d9b20d4a09f81215b4328e4b7688e545f191f 100644
--- a/browser/actors/ContextMenuChild.sys.mjs
+++ b/browser/actors/ContextMenuChild.sys.mjs
@@ -46,6 +46,10 @@ export class ContextMenuChild extends JSWindowActorChild {
 
   receiveMessage(message) {
     switch (message.name) {
+      case "ContextMenu:GetSelection": {
+        return lazy.SelectionUtils.getSelectionDetails(this.contentWindow);
+      }
+
       case "ContextMenu:GetFrameTitle": {
         let target = lazy.ContentDOMReference.resolve(
           message.data.targetIdentifier
