diff --git a/build/moz.build b/build/moz.build
index 60cf9a852efa695e09a324d8b2f291c8b43710d6..274d1850fb336a580e5658265853d7624c161ae8 100644
--- a/build/moz.build
+++ b/build/moz.build
@@ -92,7 +92,7 @@ if CONFIG["MOZ_APP_BASENAME"]:
         if CONFIG[var]:
             appini_defines[var] = True
 
-    appini_defines["MOZ_APPUPDATE_HOST"] = "aus5.mozilla.org"
+    appini_defines["MOZ_APPUPDATE_HOST"] = "update.astian.org"
     if CONFIG["MOZ_APPUPDATE_HOST"]:
         appini_defines["MOZ_APPUPDATE_HOST"] = CONFIG["MOZ_APPUPDATE_HOST"]
 
