diff --git a/browser/themes/osx/jar.mn b/browser/themes/osx/jar.mn
index 01e8a1036e2a655dc78a4d2734606cc72b1d0eee..53396b10445602506470af54ee2da33457c171bc 100644
--- a/browser/themes/osx/jar.mn
+++ b/browser/themes/osx/jar.mn
@@ -5,7 +5,7 @@
 browser.jar:
 % skin browser classic/1.0 %skin/classic/browser/
 #include ../shared/jar.inc.mn
-  skin/classic/browser/browser.css
+* skin/classic/browser/browser.css
   skin/classic/browser/customizableui/panelUI.css    (customizableui/panelUI.css)
   skin/classic/browser/downloads/allDownloadsView.css (downloads/allDownloadsView.css)
   skin/classic/browser/downloads/downloads.css              (downloads/downloads.css)
