diff --git a/browser/modules/HomePage.sys.mjs b/browser/modules/HomePage.sys.mjs
index b681ccd00e3976a2f15f2b5035a6e8a84df227c4..e4fd2247f0475fc79543d3b888565dc7e275a8e3 100644
--- a/browser/modules/HomePage.sys.mjs
+++ b/browser/modules/HomePage.sys.mjs
@@ -15,7 +15,7 @@ ChromeUtils.defineESModuleGetters(lazy, {
 });
 
 const kPrefName = "browser.startup.homepage";
-const kDefaultHomePage = "about:home";
+const kDefaultHomePage = "about:newtab";
 const kExtensionControllerPref =
   "browser.startup.homepage_override.extensionControlled";
 const kHomePageIgnoreListId = "homepage-urls";
