diff --git a/browser/modules/HomePage.sys.mjs b/browser/modules/HomePage.sys.mjs
--- a/browser/modules/HomePage.sys.mjs
+++ b/browser/modules/HomePage.sys.mjs
@@ -7,2 +7,3 @@
 ChromeUtils.defineESModuleGetters(lazy, {
+  AboutNewTab: "resource:///modules/AboutNewTab.sys.mjs",
   CustomizableUI:
@@ -120,1 +121,8 @@ export let HomePage = {
+    if (
+      ["about:home", "about:newtab"].includes(homePages) &&
+      lazy.AboutNewTab.newTabURL != "about:newtab"
+    ) {
+      homePages = lazy.AboutNewTab.newTabURL;
+    }
+
     return homePages;
