diff --git a/browser/modules/HomePage.sys.mjs b/browser/modules/HomePage.sys.mjs
index 762962c7ca610905f738440d6cd06461f99c2bd9..5c073e974d3d56e275ac4c466d16847492197f53 100644
--- a/browser/modules/HomePage.sys.mjs
+++ b/browser/modules/HomePage.sys.mjs
@@ -15,7 +15,7 @@ ChromeUtils.defineESModuleGetters(lazy, {
 });
 
 const kPrefName = "browser.startup.homepage";
-const kDefaultHomePage = "about:home";
+const kDefaultHomePage = "about:newtab";
 const kExtensionControllerPref =
   "browser.startup.homepage_override.extensionControlled";
 const kHomePageIgnoreListId = "homepage-urls";
