diff --git a/browser/base/content/appmenu-viewcache.inc.xhtml b/browser/base/content/appmenu-viewcache.inc.xhtml
index 00c4b3b071..3509b8e98d 100644
--- a/browser/base/content/appmenu-viewcache.inc.xhtml
+++ b/browser/base/content/appmenu-viewcache.inc.xhtml
@@ -62,6 +62,10 @@
                      data-l10n-id="appmenuitem-new-private-window"
                      key="key_privatebrowsing"
                      command="Tools:PrivateBrowsing"/>
+      <toolbarbutton id="appMenu-new-tor-window-button"
+                     class="subviewbutton"
+                     data-l10n-id="appmenuitem-new-tor-window"
+                     command="Tools:TorBrowsing"/>
       <toolbarseparator/>
       <toolbarbutton id="appMenu-bookmarks-button"
                      class="subviewbutton subviewbutton-nav"
