diff --git a/browser/base/content/appmenu-viewcache.inc.xhtml b/browser/base/content/appmenu-viewcache.inc.xhtml
index 4f1a7223d257d1e98fec2cda3849b66b45eebd3c..f49309be275aa8e86ff66db97d877185bd11e656 100644
--- a/browser/base/content/appmenu-viewcache.inc.xhtml
+++ b/browser/base/content/appmenu-viewcache.inc.xhtml
@@ -67,6 +67,10 @@
                      data-l10n-id="appmenuitem-new-private-window"
                      key="key_privatebrowsing"
                      command="Tools:PrivateBrowsing"/>
+      <toolbarbutton id="appMenu-new-tor-window-button"
+                     class="subviewbutton"
+                     data-l10n-id="appmenuitem-new-tor-window"
+                     command="Tools:TorBrowsing"/>
       <toolbarseparator/>
       <toolbarbutton id="appMenu-bookmarks-button"
                      class="subviewbutton subviewbutton-nav"
