diff --git a/browser/base/content/appmenu-viewcache.inc.xhtml b/browser/base/content/appmenu-viewcache.inc.xhtml
index fcca071a29dbe084ccdbe01d1d1c605748269ed7..8264bc1b608105cc03a7703016f3bc0711031e29 100644
--- a/browser/base/content/appmenu-viewcache.inc.xhtml
+++ b/browser/base/content/appmenu-viewcache.inc.xhtml
@@ -67,6 +67,10 @@
                      data-l10n-id="appmenuitem-new-private-window"
                      key="key_privatebrowsing"
                      command="Tools:PrivateBrowsing"/>
+      <toolbarbutton id="appMenu-new-tor-window-button"
+                     class="subviewbutton"
+                     data-l10n-id="appmenuitem-new-tor-window"
+                     command="Tools:TorBrowsing"/>
       <toolbarseparator/>
       <toolbarbutton id="appMenu-bookmarks-button"
                      class="subviewbutton subviewbutton-nav"
