diff --git a/browser/base/content/browser-siteIdentity.js b/browser/base/content/browser-siteIdentity.js
index 16b222280f856e697f11fee76798cc2afe0bc239..5b3ee911fa57f17b0872909e63bb4c7356de6ada 100644
--- a/browser/base/content/browser-siteIdentity.js
+++ b/browser/base/content/browser-siteIdentity.js
@@ -986,10 +986,13 @@ var gIdentityHandler = {
     }
 
     // Hide the shield icon if it is a chrome page.
-    gProtectionsHandler._trackingProtectionIconContainer.classList.toggle(
-      "chromeUI",
-      this._isSecureInternalUI
-    );
+    // Midori: The customized protection UI may omit this container.
+    if (gProtectionsHandler._trackingProtectionIconContainer) {
+      gProtectionsHandler._trackingProtectionIconContainer.classList.toggle(
+        "chromeUI",
+        this._isSecureInternalUI
+      );
+    }
   },
 
   /**
