diff --git a/browser/base/content/browser-sets.inc.xhtml b/browser/base/content/browser-sets.inc.xhtml
index 8871fc984acccfd9c54c7aef9f91934d81ba00c0..b55054f3b3b8936b468691fc2dfb516a5c95fb7b 100644
--- a/browser/base/content/browser-sets.inc.xhtml
+++ b/browser/base/content/browser-sets.inc.xhtml
@@ -103,6 +103,7 @@
     <command id="Tools:ClassicWindow" />
     <command id="Tools:Sanitize" />
     <command id="Tools:PrivateBrowsing" />
+    <command id="Tools:TorBrowsing" />
     <command id="Browser:Screenshot" />
     <command id="History:UndoCloseTab" data-l10n-args='{"tabCount": 1}' />
     <command id="History:UndoCloseWindow" />
