diff --git a/browser/base/content/browser-sets.inc.xhtml b/browser/base/content/browser-sets.inc.xhtml
index 65ee617216d8a69f29b06fb48078f53a42bd9da7..efc68478021bd545cb9faa3960dce0ac41cc7d89 100644
--- a/browser/base/content/browser-sets.inc.xhtml
+++ b/browser/base/content/browser-sets.inc.xhtml
@@ -106,6 +106,7 @@
     <command id="Tools:ClassicWindow" />
     <command id="Tools:Sanitize" />
     <command id="Tools:PrivateBrowsing" />
+    <command id="Tools:TorBrowsing" />
     <command id="Browser:Screenshot" />
     <command id="History:UndoCloseTab" data-l10n-args='{"tabCount": 1}' />
     <command id="History:UndoCloseWindow" />
