From 35e80700ed922873aa1b4b6b662bfbff95bcd747 Mon Sep 17 00:00:00 2001
From: Alfonso Hernandez <alfonso.hernandez@astian.org>
Date: Sat, 6 Dec 2025 23:38:06 -0500
Subject: [PATCH] init msidebar

---
 browser/base/content/browser.xhtml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/browser/base/content/browser.xhtml b/browser/base/content/browser.xhtml
index 246d23a779..f43cb139d5 100644
--- a/browser/base/content/browser.xhtml
+++ b/browser/base/content/browser.xhtml
@@ -100,6 +100,7 @@
 
   <script src="chrome://browser/content/global-scripts.js"></script>
   <script src="chrome://browser/content/browser-main.js"></script>
+  <script src="chrome://browser/content/msidebar-init.js"></script>
 </head>
 <html:body xmlns="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul">
 # All sets except for popupsets (commands, keys, and stringbundles)
-- 
2.52.0

