diff --git a/toolkit/library/rust/shared/Cargo.toml b/toolkit/library/rust/shared/Cargo.toml
index 88880918cb438cc93027a01aaf71cb99e6c95964..9019c8c7c3f14601e95212617cd11f90e0397402 100644
--- a/toolkit/library/rust/shared/Cargo.toml
+++ b/toolkit/library/rust/shared/Cargo.toml
@@ -123,7 +123,7 @@ urlpattern = "0.6.0"
 urlpattern_glue = { path = "../../../../netwerk/base/urlpattern_glue" }
 uritemplate_glue = { path = "../../../../netwerk/base/uritemplate_glue" }
 
-adblock = { version = "0.12.1", features = ["full-regex-handling", "single-thread"], default-features = false }
+adblock = { version = "0.13.2", features = ["full-regex-handling", "single-thread"], default-features = false }
 content_classifier_engine = { path = "../../../components/content-classifier/content_classifier_engine" }
 
 libz-rs-sys = { version = "0.6", features = ["custom-prefix"], optional = true }
