diff --git a/browser/themes/BuiltInThemes.sys.mjs b/browser/themes/BuiltInThemes.sys.mjs
index ffad3aa4c7d5d78549dbde3effe69b15cbd1f6b2..b087662e7b6d2d132ee8ae168279a88973e440f8 100644
--- a/browser/themes/BuiltInThemes.sys.mjs
+++ b/browser/themes/BuiltInThemes.sys.mjs
@@ -40,7 +40,7 @@ class _BuiltInThemes {
   maybeInstallActiveBuiltInTheme() {
     const activeThemeID = Services.prefs.getStringPref(
       kActiveThemePref,
-      "default-theme@mozilla.org"
+      "midori-theme-jade-mist@midori.astian.org"
     );
     let activeBuiltInTheme = this.builtInThemeMap.get(activeThemeID);
 
