diff --git a/browser/themes/BuiltInThemeConfig.sys.mjs b/browser/themes/BuiltInThemeConfig.sys.mjs
index c4ff66417c43c8c56948b7de4c48cf93828a66bf..016bf798c3c6b6d7429a142f3c9cb6a4cfd3075d 100644
--- a/browser/themes/BuiltInThemeConfig.sys.mjs
+++ b/browser/themes/BuiltInThemeConfig.sys.mjs
@@ -20,30 +20,4 @@
  *   non-native appearance. See Document.forceNonNativeTheme and the
  *   (-moz-native-theme) media query.
  */
-export const BuiltInThemeConfig = new Map([
-  [
-    "firefox-compact-light@mozilla.org",
-    {
-      version: "1.3.4",
-      path: "resource://builtin-themes/light/",
-      inApp: true,
-      nonNative: true,
-    },
-  ],
-  [
-    "firefox-compact-dark@mozilla.org",
-    {
-      version: "1.3.4",
-      path: "resource://builtin-themes/dark/",
-      inApp: true,
-      nonNative: true,
-    },
-  ],
-  [
-    "firefox-alpenglow@mozilla.org",
-    {
-      version: "1.5.2",
-      path: "resource://builtin-themes/alpenglow/",
-    },
-  ],
-]);
+export const BuiltInThemeConfig = new Map([]);
