From 4ed57e5794b59926db4c55383e9ca1d0d5be927c Mon Sep 17 00:00:00 2001
From: Alfonso Hernandez <alfonso.hernandez@astian.org>
Date: Tue, 18 Nov 2025 23:11:39 -0500
Subject: [PATCH] include themes custom for midori

---
 browser/themes/BuiltInThemeConfig.sys.mjs | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/browser/themes/BuiltInThemeConfig.sys.mjs b/browser/themes/BuiltInThemeConfig.sys.mjs
index 38a7726b7c..2206c3090e 100644
--- a/browser/themes/BuiltInThemeConfig.sys.mjs
+++ b/browser/themes/BuiltInThemeConfig.sys.mjs
@@ -40,4 +40,18 @@ export const BuiltInThemeConfig = new Map([
       path: "resource://builtin-themes/alpenglow/",
     },
   ],
+  [
+    "midori-light-new@astian.org",
+  {
+      version: "1.2",
+      path: "resource://builtin-themes/midori/lightnew/",
+  },
+  ],
+  [
+   "midori-dark-new@astian.org",
+    {
+      version: "1.2",
+      path: "resource://builtin-themes/midori/darknew/",
+    },
+  ],
 ]);
-- 
2.51.1

