diff --git a/browser/app/splash.rc b/browser/app/splash.rc
index ee499e2f0edfc422e979791ab4b06e606f842ff1..55bea39ea3014a198f2e08936cae7942899fe460 100644
--- a/browser/app/splash.rc
+++ b/browser/app/splash.rc
@@ -5,6 +5,9 @@
 #include <windows.h>
 #include "nsNativeAppSupportWin.h"
 
+#define MANIFEST_PATH "firefox.exe.manifest"
+CREATEPROCESS_MANIFEST_RESOURCE_ID  RT_MANIFEST             MANIFEST_PATH
+
 IDI_APPICON ICON FIREFOX_ICO
 IDI_DOCUMENT ICON DOCUMENT_ICO
 IDI_APPLICATION ICON FIREFOX_ICO
