diff --git a/browser/components/urlbar/content/SmartbarInput.mjs b/browser/components/urlbar/content/SmartbarInput.mjs
index be897dc347f3370ddeae04006a38e28de3233a07..7b736a527e1fc503da0331921e787bf70fa561a5 100644
--- a/browser/components/urlbar/content/SmartbarInput.mjs
+++ b/browser/components/urlbar/content/SmartbarInput.mjs
@@ -1033,11 +1033,20 @@ export class SmartbarInput extends HTMLElement {
       // If we update the URI while restoring a session, set the proxyState to
       // invalid, because we don't have a valid security state to show via site
       // identity yet. See Bug 1746383.
-      valid =
-        !dueToSessionRestore &&
-        (!this.window.isBlankPageURL(uri.spec) ||
-          lazy.ExtensionUtils.isExtensionUrl(uri) ||
-          isInitialPageControlledByWebContent);
+      try {
+        const policy = Cu.getGlobalForObject(Services).WebExtensionPolicy.getByURI(uri);
+        if (policy && policy.id === 'midoritabs@astian.org') {
+          value = "";
+          valid = false;
+        }
+      } catch (e) {}
+      if (valid !== false) {
+        valid =
+          !dueToSessionRestore &&
+          (!this.window.isBlankPageURL(uri.spec) ||
+            lazy.ExtensionUtils.isExtensionUrl(uri) ||
+            isInitialPageControlledByWebContent);
+      }
     } else if (
       this.window.isInitialPage(value) &&
       lazy.BrowserUIUtils.checkEmptyPageOrigin(
