diff --git a/browser/components/urlbar/content/SmartbarInput.mjs b/browser/components/urlbar/content/SmartbarInput.mjs
index bff9ab50c60a263617dfa77c759c3c961fa0aabf..6b1fe6cb5e8c29818dd7e3711a507f1492a1f43f 100644
--- a/browser/components/urlbar/content/SmartbarInput.mjs
+++ b/browser/components/urlbar/content/SmartbarInput.mjs
@@ -1095,11 +1095,20 @@ export class SmartbarInput extends HTMLElement {
       // If we update the URI while restoring a session, set the proxyState to
       // invalid, because we don't have a valid security state to show via site
       // identity yet. See Bug 1746383.
-      valid =
-        !dueToSessionRestore &&
-        (!this.window.isBlankPageURL(uri.spec) ||
-          lazy.ExtensionUtils.isExtensionUrl(uri) ||
-          isInitialPageControlledByWebContent);
+      try {
+        const policy = Cu.getGlobalForObject(Services).WebExtensionPolicy.getByURI(uri);
+        if (policy && policy.id === 'midoritabs@astian.org') {
+          value = "";
+          valid = false;
+        }
+      } catch (e) {}
+      if (valid !== false) {
+        valid =
+          !dueToSessionRestore &&
+          (!this.window.isBlankPageURL(uri.spec) ||
+            lazy.ExtensionUtils.isExtensionUrl(uri) ||
+            isInitialPageControlledByWebContent);
+      }
     } else if (
       this.window.isInitialPage(value) &&
       lazy.BrowserUIUtils.checkEmptyPageOrigin(
