diff --git a/browser/components/urlbar/content/SmartbarInput.mjs b/browser/components/urlbar/content/SmartbarInput.mjs
index 089dc6f71bc9577b2fcbea6376d955c7f780ef91..3533c447fa66f6a7521691d3ccc4f957cf27dcc7 100644
--- a/browser/components/urlbar/content/SmartbarInput.mjs
+++ b/browser/components/urlbar/content/SmartbarInput.mjs
@@ -1173,11 +1173,20 @@ ${
       // If we update the URI while restoring a session, set the proxyState to
       // invalid, because we don't have a valid security state to show via site
       // identity yet. See Bug 1746383.
-      valid =
-        !dueToSessionRestore &&
-        (!this.window.isBlankPageURL(uri.spec) ||
-          lazy.ExtensionUtils.isExtensionUrl(uri) ||
-          isInitialPageControlledByWebContent);
+      try {
+        const policy = Cu.getGlobalForObject(Services).WebExtensionPolicy.getByURI(uri);
+        if (policy && policy.id === 'midoritabs@astian.org') {
+          value = "";
+          valid = false;
+        }
+      } catch (e) {}
+      if (valid !== false) {
+        valid =
+          !dueToSessionRestore &&
+          (!this.window.isBlankPageURL(uri.spec) ||
+            lazy.ExtensionUtils.isExtensionUrl(uri) ||
+            isInitialPageControlledByWebContent);
+      }
     } else if (
       this.window.isInitialPage(value) &&
       lazy.BrowserUIUtils.checkEmptyPageOrigin(
