diff --git a/browser/components/urlbar/content/UrlbarInput.mjs b/browser/components/urlbar/content/UrlbarInput.mjs
index 24ad0ee89191831c67442b31973f4ecbff445043..beb27a236056f5d083eb7d107d88b8753399b740 100644
--- a/browser/components/urlbar/content/UrlbarInput.mjs
+++ b/browser/components/urlbar/content/UrlbarInput.mjs
@@ -849,11 +849,20 @@ export class UrlbarInput extends HTMLElement {
       // If we update the URI while restoring a session, set the proxyState to
       // invalid, because we don't have a valid security state to show via site
       // identity yet. See Bug 1746383.
-      valid =
-        !dueToSessionRestore &&
-        (!this.#canHandleAsBlankPage(uri.spec) ||
-          lazy.ExtensionUtils.isExtensionUrl(uri) ||
-          isInitialPageControlledByWebContent);
+      try {
+        const policy = Cu.getGlobalForObject(Services).WebExtensionPolicy.getByURI(uri);
+        if (policy && policy.id === 'midoritabs@astian.org') {
+          value = "";
+          valid = false;
+        }
+      } catch (e) {}
+      if (valid !== false) {
+        valid =
+          !dueToSessionRestore &&
+          (!this.#canHandleAsBlankPage(uri.spec) ||
+            lazy.ExtensionUtils.isExtensionUrl(uri) ||
+            isInitialPageControlledByWebContent);
+      }
     } else if (
       this.window.isInitialPage(value) &&
       lazy.BrowserUIUtils.checkEmptyPageOrigin(
