diff --git a/toolkit/themes/shared/desktop-jar.inc.mn b/toolkit/themes/shared/desktop-jar.inc.mn
index c8e6055a804f586c2c2dad0b836600e67222fcb4..f8337d993e3d7aba44b716dd6a8cc08155595b98 100644
--- a/toolkit/themes/shared/desktop-jar.inc.mn
+++ b/toolkit/themes/shared/desktop-jar.inc.mn
@@ -149,6 +149,7 @@
   skin/classic/global/narrate/slow.svg                     (../../shared/narrate/slow.svg)
   skin/classic/global/narrate/headphone.svg                (../../shared/narrate/headphone.svg)
   skin/classic/global/narrate/headphone-active.svg         (../../shared/narrate/headphone-active.svg)
+  skin/classic/global/pictureinpicture/midori-pip.css      (../../shared/pictureinpicture/midori-pip.css)
   skin/classic/global/pictureinpicture/player.css          (../../shared/pictureinpicture/player.css)
   skin/classic/global/pictureinpicture/texttracks.css           (../../shared/pictureinpicture/texttracks.css)
   skin/classic/global/reader/RM-Type-Controls-24x24.svg    (../../shared/reader/RM-Type-Controls-24x24.svg)
