diff --git a/toolkit/themes/shared/desktop-jar.inc.mn b/toolkit/themes/shared/desktop-jar.inc.mn
index b89a7a1c91eb07c58cbd06bebd11cc8f53312dcf..f44a4fa2b757f76d95f7d38b71bfa74dc230c91f 100644
--- a/toolkit/themes/shared/desktop-jar.inc.mn
+++ b/toolkit/themes/shared/desktop-jar.inc.mn
@@ -160,6 +160,7 @@
   skin/classic/global/narrate/slow.svg                     (../../shared/narrate/slow.svg)
   skin/classic/global/narrate/headphone.svg                (../../shared/narrate/headphone.svg)
   skin/classic/global/narrate/headphone-active.svg         (../../shared/narrate/headphone-active.svg)
+  skin/classic/global/pictureinpicture/midori-pip.css      (../../shared/pictureinpicture/midori-pip.css)
   skin/classic/global/pictureinpicture/player.css          (../../shared/pictureinpicture/player.css)
   skin/classic/global/pictureinpicture/texttracks.css           (../../shared/pictureinpicture/texttracks.css)
   skin/classic/global/reader/RM-Type-Controls-24x24.svg    (../../shared/reader/RM-Type-Controls-24x24.svg)
