From 4826ca2b1a7b19bbcdfc535f208e82ad52fe4874 Mon Sep 17 00:00:00 2001
From: Alfonso Hernandez <alfonso.hernandez@astian.org>
Date: Tue, 18 Nov 2025 15:12:35 -0500
Subject: [PATCH] add theme light new of Midori Custom

---
 browser/themes/addons/jar.mn | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/browser/themes/addons/jar.mn b/browser/themes/addons/jar.mn
index 1397afd8e2..06dcc319c4 100644
--- a/browser/themes/addons/jar.mn
+++ b/browser/themes/addons/jar.mn
@@ -15,3 +15,11 @@ browser.jar:
   content/builtin-themes/light/preview.svg         (firefox-light-dark/light-preview.svg)
   content/builtin-themes/light/icon.svg            (firefox-light-dark/light-icon.svg)
   content/builtin-themes/light/manifest.json       (light/manifest.json)
+
+  content/builtin-themes/midori/lightnew                     (midori/lightnew/*.svg)
+  content/builtin-themes/midori/lightnew/experiment.css      (midori/lightnew/experiment.css)
+  content/builtin-themes/midori/lightnew/manifest.json       (midori/lightnew/manifest.json)
+
+  content/builtin-themes/midori/darknew                      (midori/darknew/*.svg)
+  content/builtin-themes/midori/darknew/experiment.css       (midori/darknew/experiment.css)
+  content/builtin-themes/midori/darknew/manifest.json        (midori/darknew/manifest.json)
-- 
2.51.1

