diff --git a/browser/themes/shared/jar.inc.mn b/browser/themes/shared/jar.inc.mn
index 3a12a7a33c33cec463b3d38647bdf647827784f8..cb415bef3203fc498f5aeee63bf6b569d92e9c2c 100644
--- a/browser/themes/shared/jar.inc.mn
+++ b/browser/themes/shared/jar.inc.mn
@@ -18,7 +18,7 @@
   skin/classic/browser/aiWindowSidebar.css                     (../shared/aiWindowSidebar.css)
   skin/classic/browser/blockedSite.css                         (../shared/blockedSite.css)
   skin/classic/browser/browser-colors.css                      (../shared/browser-colors.css)
-  skin/classic/browser/browser-shared.css                      (../shared/browser-shared.css)
+* skin/classic/browser/browser-shared.css                      (../shared/browser-shared.css)
   skin/classic/browser/formautofill-notification.css           (../shared/formautofill-notification.css)
   skin/classic/browser/identity-credential-notification.css    (../shared/identity-credential-notification.css)
   skin/classic/browser/menupanel.css                           (../shared/menupanel.css)
