diff --git a/toolkit/components/pdfjs/jar.mn b/toolkit/components/pdfjs/jar.mn
index 9e087510e65007674a5c7e6514aa141fbbbd5b79..0f6242c6885348ddb03de8e3038e2eb6a55f3f09 100644
--- a/toolkit/components/pdfjs/jar.mn
+++ b/toolkit/components/pdfjs/jar.mn
@@ -29,6 +29,7 @@ pdfjs.jar:
  content/PdfjsContextMenu.sys.mjs (content/PdfjsContextMenu.sys.mjs)
  content/web/viewer.html (content/web/viewer.html)
  content/web/viewer.css (content/web/viewer.css)
+ content/web/midori-pdfjs.css (content/web/midori-pdfjs.css)
  content/web/viewer.mjs (content/web/viewer.mjs)
  content/web/debugger.css (content/web/debugger.css)
  content/web/debugger.mjs (content/web/debugger.mjs)
