diff --git a/toolkit/components/pdfjs/jar.mn b/toolkit/components/pdfjs/jar.mn
index 2b5a220dbe4fcc5218da8583332ad4d9f5df26c7..f3056a58b5021788716d4cd0bf50a55cf497631a 100644
--- a/toolkit/components/pdfjs/jar.mn
+++ b/toolkit/components/pdfjs/jar.mn
@@ -30,6 +30,7 @@ pdfjs.jar:
  content/PdfjsContextMenu.sys.mjs (content/PdfjsContextMenu.sys.mjs)
  content/web/viewer.html (content/web/viewer.html)
  content/web/viewer.css (content/web/viewer.css)
+ content/web/midori-pdfjs.css (content/web/midori-pdfjs.css)
  content/web/viewer.mjs (content/web/viewer.mjs)
  content/web/debugger.css (content/web/debugger.css)
  content/web/debugger.mjs (content/web/debugger.mjs)
