diff --git a/browser/components/about/components.conf b/browser/components/about/components.conf
--- a/browser/components/about/components.conf
+++ b/browser/components/about/components.conf
@@ -7,6 +7,7 @@
     'aitab',
     'asrouter',
     'blocked',
+    'center',
     'certerror',
     'downloads',
     'framecrashed',
@@ -14,6 +15,7 @@
     'logins',
     'loginsimportreport',
     'firefoxview',
+    'midorilink',
     'messagepreview',
     'opentabs',
     'policies',
@@ -28,6 +30,7 @@
     'robots',
     'sessionrestore',
     'settings',
+    'setup',
     'smartformfillreview',
     'smartwindowtasks',
     'tabcrashed',
