diff --git a/browser/components/about/components.conf b/browser/components/about/components.conf
index d5b79f6dfa98820b2e6f22102fa63cb362ca6c8c..5f1bd39e53ba3de9113a34b3518f393073299a54 100644
--- a/browser/components/about/components.conf
+++ b/browser/components/about/components.conf
@@ -6,6 +6,7 @@ pages = [
     'aichatcontent',
     'asrouter',
     'blocked',
+    'center',
     'certerror',
     'downloads',
     'framecrashed',
@@ -27,6 +28,7 @@ pages = [
     'robots',
     'sessionrestore',
     'settings',
+    'setup',
     'smartwindowtasks',
     'tabcrashed',
     'unloads',
