diff --git a/browser/components/about/components.conf b/browser/components/about/components.conf
index 669c6912f636ee29703bec015ce54fbd5fed90ad..35215ec4022c9f529236a168dbeb2d942c8f04ce 100644
--- a/browser/components/about/components.conf
+++ b/browser/components/about/components.conf
@@ -6,6 +6,7 @@ pages = [
     'aichatcontent',
     'asrouter',
     'blocked',
+    'center',
     'certerror',
     'downloads',
     'framecrashed',
@@ -26,6 +27,7 @@ pages = [
     'robots',
     'sessionrestore',
     'settings',
+    'setup',
     'tabcrashed',
     'unloads',
     'welcome',
