diff --git a/browser/themes/linux/jar.mn b/browser/themes/linux/jar.mn
index 0a6465a26f3938008c836ff62123a80ee1d68688..e9f6e8d7354c52c074f24306dcf6aaafd99ca5f4 100644
--- a/browser/themes/linux/jar.mn
+++ b/browser/themes/linux/jar.mn
@@ -5,7 +5,7 @@
 browser.jar:
 % skin browser classic/1.0 %skin/classic/browser/
 #include ../shared/jar.inc.mn
-  skin/classic/browser/browser.css
+* skin/classic/browser/browser.css
   skin/classic/browser/contextmenu.css                (../shared/contextmenu.css)
   skin/classic/browser/monitor-base.png
   skin/classic/browser/monitor-border.png
