--- a/security/manager/ssl/CredentialManagerSecret.cpp
+++ b/security/manager/ssl/CredentialManagerSecret.cpp
@@ -79,9 +79,9 @@
   return NS_OK;
 }
 
-nsresult CredentialManagerSecret::RetrieveSecret(
-    const nsACString& aLabel,
-    /* out */ nsACString& aSecret) {
+nsresult CredentialManagerSecret::RetrieveSecret(const nsACString& aLabel,
+                                                 /* out */ nsACString& aSecret,
+                                                 bool /*aPreventPrompt*/) {
   aSecret.Truncate();
   PCREDENTIALA pcred_raw = nullptr;
   const nsCString& label = PromiseFlatCString(aLabel);
