--- a/security/manager/ssl/CredentialManagerSecret.h
+++ b/security/manager/ssl/CredentialManagerSecret.h
@@ -14,7 +14,8 @@
   CredentialManagerSecret() = default;
 
   virtual nsresult RetrieveSecret(const nsACString& label,
-                                  /* out */ nsACString& secret) override;
+                                  /* out */ nsACString& secret,
+                                  bool aPreventPrompt = false) override;
   virtual nsresult StoreSecret(const nsACString& secret,
                                const nsACString& label) override;
   virtual nsresult DeleteSecret(const nsACString& label) override;
