--- a/security/manager/ssl/KeychainSecret.h
+++ b/security/manager/ssl/KeychainSecret.h
@@ -36,7 +36,8 @@
   KeychainSecret();
 
   virtual nsresult RetrieveSecret(const nsACString& label,
-                                  /* out */ nsACString& secret) override;
+                                  /* out */ nsACString& secret,
+                                  bool aPreventPrompt = false) override;
   virtual nsresult StoreSecret(const nsACString& secret,
                                const nsACString& label) override;
   virtual nsresult DeleteSecret(const nsACString& label) override;
