--- a/security/manager/ssl/KeychainSecret.cpp
+++ b/security/manager/ssl/KeychainSecret.cpp
@@ -123,7 +123,8 @@
 }
 
 nsresult KeychainSecret::RetrieveSecret(const nsACString& aLabel,
-                                        /* out */ nsACString& aSecret) {
+                                        /* out */ nsACString& aSecret,
+                                        bool aPreventPrompt) {
   // To retrieve a secret, we create a CFDictionary of the form:
   // { class: generic password,
   //   account: the given label,
@@ -133,16 +134,22 @@
   // This searches for and returns the attributes and data for the secret
   // matching the given label. We then extract the data (i.e. the secret) and
   // return it.
-  const CFStringRef keys[] = {kSecClass, kSecAttrAccount, kSecMatchLimit,
-                              kSecReturnAttributes, kSecReturnData};
+  const CFStringRef keys[] = {kSecClass,      kSecAttrAccount,
+                              kSecMatchLimit, kSecReturnAttributes,
+                              kSecReturnData, kSecUseAuthenticationUI};
   ScopedCFType<CFStringRef> label(MozillaStringToCFString(aLabel));
   if (!label) {
     MOZ_LOG(gKeychainSecretLog, LogLevel::Debug,
             ("MozillaStringToCFString failed"));
     return NS_ERROR_FAILURE;
   }
-  const void* values[] = {kSecClassGenericPassword, label.get(),
-                          kSecMatchLimitOne, kCFBooleanTrue, kCFBooleanTrue};
+  const void* values[] = {kSecClassGenericPassword,
+                          label.get(),
+                          kSecMatchLimitOne,
+                          kCFBooleanTrue,
+                          kCFBooleanTrue,
+                          aPreventPrompt ? kSecUseAuthenticationUIFail
+                                         : kSecUseAuthenticationUIAllow};
   static_assert(std::size(keys) == std::size(values),
                 "mismatched SecItemCopyMatching key/value array sizes");
   ScopedCFType<CFDictionaryRef> searchDictionary(CFDictionaryCreate(
@@ -151,6 +158,10 @@
   CFTypeRef item;
   // https://developer.apple.com/documentation/security/1398306-secitemcopymatching
   OSStatus osrv = SecItemCopyMatching(searchDictionary.get(), &item);
+  if (aPreventPrompt &&
+      (osrv == errSecInteractionNotAllowed || osrv == errSecAuthFailed)) {
+    return NS_ERROR_ABORT;
+  }
   if (osrv == errSecItemNotFound) {
     MOZ_LOG(gKeychainSecretLog, LogLevel::Debug,
             ("Key not found in key store"));
