--- a/security/manager/ssl/LibSecret.h
+++ b/security/manager/ssl/LibSecret.h
@@ -16,7 +16,8 @@
   LibSecret();
 
   virtual nsresult RetrieveSecret(const nsACString& label,
-                                  /* out */ nsACString& secret) override;
+                                  /* out */ nsACString& secret,
+                                  bool aPreventPrompt = false) override;
   virtual nsresult StoreSecret(const nsACString& secret,
                                const nsACString& label) override;
   virtual nsresult DeleteSecret(const nsACString& label) override;
