--- a/security/manager/ssl/NSSKeyStore.h
+++ b/security/manager/ssl/NSSKeyStore.h
@@ -14,15 +14,17 @@
   NSSKeyStore();
 
   virtual nsresult RetrieveSecret(const nsACString& aLabel,
-                                  /* out */ nsACString& aSecret) override;
+                                  /* out */ nsACString& aSecret,
+                                  bool aPreventPrompt = false) override;
   virtual nsresult StoreSecret(const nsACString& secret,
                                const nsACString& label) override;
   virtual nsresult DeleteSecret(const nsACString& label) override;
   virtual nsresult EncryptDecrypt(const nsACString& label,
                                   const std::vector<uint8_t>& inBytes,
-                                  std::vector<uint8_t>& outBytes,
-                                  bool encrypt) override;
-  virtual nsresult SecretAvailable(const nsACString& label) override;
+                                  std::vector<uint8_t>& outBytes, bool encrypt,
+                                  bool aPreventPrompt = false) override;
+  virtual nsresult SecretAvailable(const nsACString& label,
+                                   bool aPreventPrompt = false) override;
   virtual ~NSSKeyStore();
 
  private:
