--- a/security/manager/ssl/NSSKeyStore.cpp
+++ b/security/manager/ssl/NSSKeyStore.cpp
@@ -100,9 +100,15 @@
   return NS_OK;
 }
 
-nsresult NSSKeyStore::SecretAvailable(const nsACString& aLabel) {
+nsresult NSSKeyStore::SecretAvailable(const nsACString& aLabel,
+                                      bool aPreventPrompt) {
   if (!mSlot) {
     return NS_ERROR_NOT_AVAILABLE;
+  }
+
+  if (aPreventPrompt && PK11_NeedLogin(mSlot.get()) &&
+      !PK11_IsLoggedIn(mSlot.get(), nullptr)) {
+    return NS_ERROR_ABORT;
   }
 
   UniquePK11SymKey symKey(PK11_ListFixedKeysInSlot(
@@ -117,8 +123,13 @@
 nsresult NSSKeyStore::EncryptDecrypt(const nsACString& aLabel,
                                      const std::vector<uint8_t>& inBytes,
                                      std::vector<uint8_t>& outBytes,
-                                     bool encrypt) {
-  NS_ENSURE_STATE(mSlot);
+                                     bool encrypt, bool aPreventPrompt) {
+  NS_ENSURE_STATE(mSlot);
+
+  if (aPreventPrompt && PK11_NeedLogin(mSlot.get()) &&
+      !PK11_IsLoggedIn(mSlot.get(), nullptr)) {
+    return NS_ERROR_ABORT;
+  }
 
   UniquePK11SymKey symKey(PK11_ListFixedKeysInSlot(
       mSlot.get(), const_cast<char*>(PromiseFlatCString(aLabel).get()),
@@ -132,8 +143,14 @@
 }
 
 nsresult NSSKeyStore::RetrieveSecret(const nsACString& aLabel,
-                                     /* out */ nsACString& aSecret) {
-  NS_ENSURE_STATE(mSlot);
+                                     /* out */ nsACString& aSecret,
+                                     bool aPreventPrompt) {
+  NS_ENSURE_STATE(mSlot);
+
+  if (aPreventPrompt && PK11_NeedLogin(mSlot.get()) &&
+      !PK11_IsLoggedIn(mSlot.get(), nullptr)) {
+    return NS_ERROR_ABORT;
+  }
 
   UniquePK11SymKey symKey(PK11_ListFixedKeysInSlot(
       mSlot.get(), const_cast<char*>(PromiseFlatCString(aLabel).get()),
