--- a/security/manager/ssl/OSKeyStore.cpp
+++ b/security/manager/ssl/OSKeyStore.cpp
@@ -60,11 +60,14 @@
 
 #if defined(XP_MACOSX)
   mKs.reset(new KeychainSecret());
+  mIsOSBacked = true;
 #elif defined(XP_WIN)
   mKs.reset(new CredentialManagerSecret());
+  mIsOSBacked = true;
 #elif defined(MOZ_WIDGET_GTK)
   if (NS_SUCCEEDED(MaybeLoadLibSecret())) {
     mKs.reset(new LibSecret());
+    mIsOSBacked = true;
   } else {
     mKs.reset(new NSSKeyStore());
   }
@@ -74,6 +77,19 @@
 
   (void)NS_CreateBackgroundTaskQueue(
       "OSKeyStore", getter_AddRefs(mBackgroundSerialEventTarget));
+}
+
+NS_IMETHODIMP OSKeyStore::GetIsOSBacked(bool* aIsOSBacked) {
+  MOZ_ASSERT(NS_IsMainThread());
+  NS_ENSURE_ARG_POINTER(aIsOSBacked);
+  *aIsOSBacked = mIsOSBacked;
+  return NS_OK;
+}
+
+NS_IMETHODIMP OSKeyStore::GetSupportsNonInteractiveAccess(bool* aSupported) {
+  NS_ENSURE_ARG_POINTER(aSupported);
+  *aSupported = true;
+  return NS_OK;
 }
 
 static nsresult GenerateRandom(std::vector<uint8_t>& r) {
@@ -95,9 +111,10 @@
 }
 
 nsresult OSKeyStore::SecretAvailable(const nsACString& aLabel,
-                                     /* out */ bool* aAvailable) {
+                                     /* out */ bool* aAvailable,
+                                     bool aPreventPrompt) {
   NS_ENSURE_STATE(mKs);
-  nsresult rv = mKs->SecretAvailable(aLabel);
+  nsresult rv = mKs->SecretAvailable(aLabel, aPreventPrompt);
   *aAvailable = false;
   if (rv == nsresult::NS_ERROR_NOT_AVAILABLE) {
     // This indicates that there was no such entry in the keystore. Returning
@@ -187,13 +204,14 @@
 
 nsresult OSKeyStore::EncryptBytes(const nsACString& aLabel,
                                   const std::vector<uint8_t>& aInBytes,
-                                  /*out*/ nsACString& aEncryptedBase64Text) {
+                                  /*out*/ nsACString& aEncryptedBase64Text,
+                                  bool aPreventPrompt) {
   NS_ENSURE_STATE(mKs);
 
   aEncryptedBase64Text.Truncate();
   std::vector<uint8_t> outBytes;
-  nsresult rv =
-      mKs->EncryptDecrypt(aLabel, aInBytes, outBytes, Cipher::Encrypt);
+  nsresult rv = mKs->EncryptDecrypt(aLabel, aInBytes, outBytes, Cipher::Encrypt,
+                                    aPreventPrompt);
   if (NS_FAILED(rv)) {
     return rv;
   }
@@ -213,7 +231,8 @@
 nsresult OSKeyStore::DecryptBytes(const nsACString& aLabel,
                                   const nsACString& aEncryptedBase64Text,
                                   /*out*/ uint32_t* outLen,
-                                  /*out*/ uint8_t** outBytes) {
+                                  /*out*/ uint8_t** outBytes,
+                                  bool aPreventPrompt) {
   NS_ENSURE_STATE(mKs);
   NS_ENSURE_ARG_POINTER(outLen);
   NS_ENSURE_ARG_POINTER(outBytes);
@@ -229,7 +248,7 @@
   const std::vector<uint8_t> ciphertextBytes(tmp, tmp + ciphertext.Length());
   std::vector<uint8_t> plaintextBytes;
   rv = mKs->EncryptDecrypt(aLabel, ciphertextBytes, plaintextBytes,
-                           Cipher::Decrypt);
+                           Cipher::Decrypt, aPreventPrompt);
   if (NS_FAILED(rv)) {
     return rv;
   }
@@ -311,9 +330,9 @@
 
 void BackgroundSecretAvailable(const nsACString& aLabel,
                                RefPtr<Promise>& aPromise,
-                               RefPtr<OSKeyStore> self) {
+                               RefPtr<OSKeyStore> self, bool aPreventPrompt) {
   bool available = false;
-  nsresult rv = self->SecretAvailable(aLabel, &available);
+  nsresult rv = self->SecretAvailable(aLabel, &available, aPreventPrompt);
   nsCOMPtr<nsIRunnable> runnable(NS_NewRunnableFunction(
       "BackgroundSecreteAvailableOSKSResolve",
       [rv, aPromise = std::move(aPromise), available = available]() {
@@ -327,8 +346,8 @@
 }
 
 NS_IMETHODIMP
-OSKeyStore::AsyncSecretAvailable(const nsACString& aLabel, JSContext* aCx,
-                                 Promise** promiseOut) {
+OSKeyStore::AsyncSecretAvailable(const nsACString& aLabel, bool aPreventPrompt,
+                                 JSContext* aCx, Promise** promiseOut) {
   MOZ_ASSERT(NS_IsMainThread());
   if (!NS_IsMainThread()) {
     return NS_ERROR_NOT_SAME_THREAD;
@@ -348,9 +367,9 @@
 
   RefPtr<OSKeyStore> self = this;
   nsCOMPtr<nsIRunnable> runnable(NS_NewRunnableFunction(
-      "BackgroundSecretAvailable",
-      [self, promiseHandle, aLabel = nsAutoCString(aLabel)]() mutable {
-        BackgroundSecretAvailable(aLabel, promiseHandle, self);
+      "BackgroundSecretAvailable", [self, promiseHandle, aPreventPrompt,
+                                    aLabel = nsAutoCString(aLabel)]() mutable {
+        BackgroundSecretAvailable(aLabel, promiseHandle, self, aPreventPrompt);
       }));
 
   promiseHandle.forget(promiseOut);
@@ -459,9 +478,11 @@
 static void BackgroundEncryptBytes(const nsACString& aLabel,
                                    const std::vector<uint8_t>& aInBytes,
                                    RefPtr<Promise>& aPromise,
-                                   RefPtr<OSKeyStore> self) {
+                                   RefPtr<OSKeyStore> self,
+                                   bool aPreventPrompt) {
   nsAutoCString ciphertext;
-  nsresult rv = self->EncryptBytes(aLabel, aInBytes, ciphertext);
+  nsresult rv =
+      self->EncryptBytes(aLabel, aInBytes, ciphertext, aPreventPrompt);
   nsAutoString ctext;
   CopyUTF8toUTF16(ciphertext, ctext);
 
@@ -479,7 +500,8 @@
 
 NS_IMETHODIMP
 OSKeyStore::AsyncEncryptBytes(const nsACString& aLabel,
-                              const nsTArray<uint8_t>& inBytes, JSContext* aCx,
+                              const nsTArray<uint8_t>& inBytes,
+                              bool aPreventPrompt, JSContext* aCx,
                               Promise** promiseOut) {
   MOZ_ASSERT(NS_IsMainThread());
   if (!NS_IsMainThread()) {
@@ -501,11 +523,12 @@
   RefPtr<OSKeyStore> self = this;
   nsCOMPtr<nsIRunnable> runnable(NS_NewRunnableFunction(
       "BackgroundEncryptBytes",
-      [promiseHandle,
+      [promiseHandle, aPreventPrompt,
        inBytes = std::vector<uint8_t>(inBytes.Elements(),
                                       inBytes.Elements() + inBytes.Length()),
        aLabel = nsAutoCString(aLabel), self]() mutable {
-        BackgroundEncryptBytes(aLabel, inBytes, promiseHandle, self);
+        BackgroundEncryptBytes(aLabel, inBytes, promiseHandle, self,
+                               aPreventPrompt);
       }));
 
   promiseHandle.forget(promiseOut);
@@ -515,12 +538,12 @@
 
 void BackgroundDecryptBytes(const nsACString& aLabel,
                             const nsACString& aEncryptedBase64Text,
-                            RefPtr<Promise>& aPromise,
-                            RefPtr<OSKeyStore> self) {
+                            RefPtr<Promise>& aPromise, RefPtr<OSKeyStore> self,
+                            bool aPreventPrompt) {
   uint8_t* plaintext = nullptr;
   uint32_t plaintextLen = 0;
   nsresult rv = self->DecryptBytes(aLabel, aEncryptedBase64Text, &plaintextLen,
-                                   &plaintext);
+                                   &plaintext, aPreventPrompt);
   nsTArray<uint8_t> plain;
   if (plaintext) {
     MOZ_ASSERT(plaintextLen > 0);
@@ -543,7 +566,8 @@
 NS_IMETHODIMP
 OSKeyStore::AsyncDecryptBytes(const nsACString& aLabel,
                               const nsACString& aEncryptedBase64Text,
-                              JSContext* aCx, Promise** promiseOut) {
+                              bool aPreventPrompt, JSContext* aCx,
+                              Promise** promiseOut) {
   MOZ_ASSERT(NS_IsMainThread());
   if (!NS_IsMainThread()) {
     return NS_ERROR_NOT_SAME_THREAD;
@@ -564,11 +588,11 @@
   RefPtr<OSKeyStore> self = this;
   nsCOMPtr<nsIRunnable> runnable(NS_NewRunnableFunction(
       "BackgroundDecryptBytes",
-      [promiseHandle, self,
+      [promiseHandle, self, aPreventPrompt,
        aEncryptedBase64Text = nsAutoCString(aEncryptedBase64Text),
        aLabel = nsAutoCString(aLabel)]() mutable {
         BackgroundDecryptBytes(aLabel, aEncryptedBase64Text, promiseHandle,
-                               self);
+                               self, aPreventPrompt);
       }));
 
   promiseHandle.forget(promiseOut);
@@ -732,9 +756,10 @@
   return NS_OK;
 }
 
-nsresult AbstractOSKeyStore::SecretAvailable(const nsACString& aLabel) {
+nsresult AbstractOSKeyStore::SecretAvailable(const nsACString& aLabel,
+                                             bool aPreventPrompt) {
   nsAutoCString secret;
-  nsresult rv = RetrieveSecret(aLabel, secret);
+  nsresult rv = RetrieveSecret(aLabel, secret, aPreventPrompt);
   NS_ENSURE_SUCCESS(rv, rv);
   if (secret.Length() == 0) {
     // This should probably never happen.
@@ -747,11 +772,11 @@
 nsresult AbstractOSKeyStore::EncryptDecrypt(const nsACString& aLabel,
                                             const std::vector<uint8_t>& inBytes,
                                             std::vector<uint8_t>& outBytes,
-                                            bool encrypt) {
+                                            bool encrypt, bool aPreventPrompt) {
   nsAutoCString secret;
-  nsresult rv = RetrieveSecret(aLabel, secret);
+  nsresult rv = RetrieveSecret(aLabel, secret, aPreventPrompt);
   if (NS_FAILED(rv) || secret.Length() == 0) {
-    return NS_ERROR_FAILURE;
+    return aPreventPrompt && NS_FAILED(rv) ? rv : NS_ERROR_FAILURE;
   }
 
   uint8_t* p = BitwiseCast<uint8_t*, const char*>(secret.BeginReading());
