--- a/security/manager/ssl/OSKeyStore.h
+++ b/security/manager/ssl/OSKeyStore.h
@@ -22,7 +22,8 @@
  public:
   // Retrieve a secret with the given label.
   virtual nsresult RetrieveSecret(const nsACString& aLabel,
-                                  /* out */ nsACString& aSecret) = 0;
+                                  /* out */ nsACString& aSecret,
+                                  bool aPreventPrompt = false) = 0;
   // Store a new secret with the given label.
   virtual nsresult StoreSecret(const nsACString& secret,
                                const nsACString& label) = 0;
@@ -32,13 +33,15 @@
 
   // Returns NS_OK if the secret with the given label is available in the key
   // store, an error indicating why it was not available otherwise.
-  virtual nsresult SecretAvailable(const nsACString& label);
+  virtual nsresult SecretAvailable(const nsACString& label,
+                                   bool aPreventPrompt = false);
   // Perform encryption or decryption operation with the given secret and input
   // bytes. The output is written in outBytes. This function can make use of the
   // AesGcm class to use NSS for encryption and decryption.
   virtual nsresult EncryptDecrypt(const nsACString& label,
                                   const std::vector<uint8_t>& inBytes,
-                                  std::vector<uint8_t>& outBytes, bool encrypt);
+                                  std::vector<uint8_t>& outBytes, bool encrypt,
+                                  bool aPreventPrompt = false);
 
   size_t GetKeyByteLength() { return mKeyByteLength; }
 
@@ -74,7 +77,8 @@
   nsresult GenerateSecret(const nsACString& aLabel,
                           /* out */ nsACString& aRecoveryPhrase);
   nsresult SecretAvailable(const nsACString& aLabel,
-                           /* out */ bool* aAvailable);
+                           /* out */ bool* aAvailable,
+                           bool aPreventPrompt = false);
   nsresult RecoverSecret(const nsACString& aLabel,
                          const nsACString& aRecoveryPhrase);
   nsresult DeleteSecret(const nsACString& aLabel);
@@ -82,17 +86,20 @@
                                   /* out */ nsACString& aRecoveryPhrase);
   nsresult EncryptBytes(const nsACString& aLabel,
                         const std::vector<uint8_t>& aInBytes,
-                        /*out*/ nsACString& aEncryptedBase64Text);
+                        /*out*/ nsACString& aEncryptedBase64Text,
+                        bool aPreventPrompt = false);
   nsresult DecryptBytes(const nsACString& aLabel,
                         const nsACString& aEncryptedBase64Text,
                         /*out*/ uint32_t* outLen,
-                        /*out*/ uint8_t** outBytes);
+                        /*out*/ uint8_t** outBytes,
+                        bool aPreventPrompt = false);
 
  private:
   OSKeyStore();
   ~OSKeyStore() = default;
 
   std::unique_ptr<AbstractOSKeyStore> mKs;
+  bool mIsOSBacked = false;
   nsCOMPtr<nsISerialEventTarget> mBackgroundSerialEventTarget;
 };
 
