diff --git a/browser/components/customizableui/CustomizableUI.sys.mjs b/browser/components/customizableui/CustomizableUI.sys.mjs
index 4aca0aa0c90603e09b58395debb62875c018a8cf..13b64ccb14de777599bc1cec59161fa7a1eabcb2 100644
--- a/browser/components/customizableui/CustomizableUI.sys.mjs
+++ b/browser/components/customizableui/CustomizableUI.sys.mjs
@@ -18,6 +18,7 @@ ChromeUtils.defineESModuleGetters(lazy, {
   PanelMultiView:
     "moz-src:///browser/components/customizableui/PanelMultiView.sys.mjs",
   PrivateBrowsingUtils: "resource://gre/modules/PrivateBrowsingUtils.sys.mjs",
+  ExtensionCommon: "resource://gre/modules/ExtensionCommon.sys.mjs",
   ShortcutUtils: "resource://gre/modules/ShortcutUtils.sys.mjs",
 });
 
@@ -28,6 +29,8 @@ ChromeUtils.defineLazyGetter(lazy, "gWidgetsBundle", function () {
 });
 
 const kDefaultThemeID = "default-theme@mozilla.org";
+const kMidoriPrivacyID = "midori-protection@astian.org";
+const kMidoriVPNID = "midorivpn@astian.org";
 
 const kSpecialWidgetPfx = "customizableui-special-";
 
@@ -353,6 +356,7 @@ var CustomizableUIInternal = {
       false
     );
 
+    const { makeWidgetId } = lazy.ExtensionCommon;
     let navbarPlacements = [
       lazy.sidebarRevampEnabled ? "sidebar-button" : null,
       "back-button",
@@ -363,7 +367,9 @@ var CustomizableUIInternal = {
         : "home-button",
       "spring",
       "vertical-spacer",
+      `${makeWidgetId(kMidoriPrivacyID)}-browser-action`,
       "urlbar-container",
+      `${makeWidgetId(kMidoriVPNID)}-browser-action`,
       "spring",
       "downloads-button",
       AppConstants.MOZ_DEV_EDITION ? "developer-button" : null,
@@ -379,7 +385,6 @@ var CustomizableUIInternal = {
         overflowable: true,
         defaultPlacements: navbarPlacements,
         verticalTabsDefaultPlacements: [
-          "firefox-view-button",
           "alltabs-button",
         ],
         defaultCollapsed: false,
@@ -404,7 +409,6 @@ var CustomizableUIInternal = {
       {
         type: CustomizableUI.TYPE_TOOLBAR,
         defaultPlacements: [
-          "firefox-view-button",
           "tabbrowser-tabs",
           "new-tab-button",
           "alltabs-button",
@@ -757,18 +761,6 @@ var CustomizableUIInternal = {
       }
     }
 
-    // Add firefox-view if not present
-    if (currentVersion < 18) {
-      let tabstripPlacements =
-        gSavedState.placements[CustomizableUI.AREA_TABSTRIP];
-      if (
-        tabstripPlacements &&
-        !tabstripPlacements.includes("firefox-view-button")
-      ) {
-        tabstripPlacements.unshift("firefox-view-button");
-      }
-    }
-
     // Unified Extensions addon button migration, which puts any browser action
     // buttons in the overflow menu into the addons panel instead.
     if (currentVersion < 19) {
