diff --git a/browser/installer/windows/msix/AppxManifest.xml.in b/browser/installer/windows/msix/AppxManifest.xml.in
index 246f15595dadbeee8229db53bf9904e89f7622ae..1b97b72a3befd76e51d2fc0144934dd447b3844a 100644
--- a/browser/installer/windows/msix/AppxManifest.xml.in
+++ b/browser/installer/windows/msix/AppxManifest.xml.in
@@ -61,6 +61,7 @@
               <uap:FileType>.avif</uap:FileType>
               <uap:FileType>.htm</uap:FileType>
               <uap:FileType>.html</uap:FileType>
+              <uap:FileType>.jxl</uap:FileType>
               <uap:FileType>.pdf</uap:FileType>
               <uap:FileType>.shtml</uap:FileType>
               <uap:FileType>.xht</uap:FileType>
